ql/instruments/swaption.hpp

```diff
@@ -111,13 +125,17 @@ namespace QuantLib {
         arguments() : settlementType(Settlement::Physical) {}
         ext::shared_ptr<VanillaSwap> swap;
         Settlement::Type settlementType;
+        Settlement::Method settlementMethod;
         void validate() const;
     };
 
     //! base class for swaption engines
     class Swaption::engine
         : public GenericEngine<Swaption::arguments, Swaption::results> {};
 
+    //! check consistency of settlement type and method
+    void checkSettlementTypeAndMethodConsistency(
```

Comment:
Since I'm being picky (😄) I'd move this to `Settlement::checkTypeAndMethodConsistency`.  I can do that when I merge, if you're ok with it.

Reply:
sure, thanks!